client/network/src/block_requests.rs

```diff
@@ -136,9 +136,9 @@ impl Config {
 	/// - request timeout = 40s
 	pub fn new(id: &ProtocolId) -> Self {
 		let mut c = Config {
-			max_block_data_response: 128,
+			max_block_data_response: 64,
```

Comment:
Can you use the setter from our runtime, instead of forking the module?

Reply:
Which setter? Aren't these fields private and non-configurable?

Reply:
The config is passed here
https://github.com/chronicled/substrate/blob/master/client/network/src/service.rs#L239
to the behavior
https://github.com/chronicled/substrate/blob/master/client/network/src/service.rs#L282
and cannot be changed after